astrodb_utils/publications.py

```diff
@@ -182,38 +167,31 @@ def find_publication(
                     return False, n_pubs_found_short_date
             else:
                 return False, n_pubs_found_short
+
     if n_pubs_found == 0 and bibcode and "arXiv" in bibcode and use_ads:
         logger.debug(f"Using ADS to find alt name for {bibcode}")
-        arxiv_id = bibcode
-        arxiv_matches = ads.SearchQuery(
-            q=arxiv_id, fl=["id", "bibcode", "title", "first_author", "year", "doi"]
+        results = search_ads(
+            bibcode, query_type="arxiv"
         )
-        arxiv_matches_list = list(arxiv_matches)
-        if len(arxiv_matches_list) == 1:
-            logger.debug(f"Publication found in ADS using arxiv id: , {arxiv_id}")
-            article = arxiv_matches_list[0]
+        bibcode_alt = results[1]
```

Comment:
If search_ads fails it returns nothing, which implies results would be `None` and `results[1]` would through a KeyError or something similar. Maybe there needs to still be a check for `if results is not None`?